src/dtos/create.log.dto.ts

```diff
@@ -17,29 +19,31 @@ export class CreateLogDto {
         example: ['run'],
         description: 'What kind of log is it?',
     })
+    // each:true makes sure in the case more than one subtype is chosen they are all validated
+    @IsEnum(SubType, { each: true, message: 'Each value in subtype must be a valid enum value' })
     subtype: [];
 
     @ApiModelProperty({
         example: ['human'],
         description: 'Where did the log come from?',
     })
+    @IsEnum(Origin, { each: true, message: 'Each value in origin must be a valid enum value' })
     origin: [];
 
-    @ApiModelProperty({
-        example: new Date(),
-        description: 'Date of creation',
-    })
-    @IsDate() creationTime: Date;
+    @IsEmpty({ message: 'No Date can be given from the frontend.' })
```

Comment:
Maybe change this to something like: 'creationTime not allowed in request body.', since API calls can also be done from a place other than a frontend.